### PR TITLE
Add wdio-rerun-service entries to Webdriver.IO

### PR DESCRIPTION
**Pull request by mikesalvia**
## Proposed changes

Adds `wdio-rerun-service` to the `wdio-cli` `constants.js`, also there is a bug in the `Custom Services` page of website that points contributors to an incorrect location.

NPM link: https://www.npmjs.com/package/wdio-rerun-service
GitHub link: https://github.com/jwplayer/wdio-rerun-service
Medium link: https://medium.com/jw-player-engineering/allow-re-running-failed-tests-in-easily-in-webdriverio-813ab08ad743

[//]: # (Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.)

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [X] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)
- [X] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by mikesalvia** [comment]
@christian-bromann Is this the only place I should update the available service?

**Review by christian-bromann — CHANGES_REQUESTED** [review]
@mikesalvia please also add it to `/scripts/docs-generation/3rd-party/services.json`, thanks!

**Comment by mikesalvia** [comment]
@christian-bromann All done!

File: scripts/docs-generation/3rd-party/services.json [hunk 1]
```diff
@@ -70,5 +70,11 @@
     "title": "Novus Visual Regression Service",
     "githubUrl": "https://github.com/Jnegrier/wdio-novus-visual-regression-service",
     "npmUrl": "https://www.npmjs.com/package/wdio-novus-visual-regression-service"
+  },
+   {
+    "packageName": "wdio-rerun-service",
+    "title": "Re-run Service",
+    "githubUrl": "https://github.com/jwplayer/wdio-rerun-service",
+    "npmUrl": "https://www.npmjs.com/package/wdio-rerun-service"
```
Comment (christian-bromann) [inline scripts/docs-generation/3rd-party/services.json:78]:
```suggestion
  },
  {
    "packageName": "wdio-rerun-service",
    "title": "Re-run Service",
    "githubUrl": "https://github.com/jwplayer/wdio-rerun-service",
    "npmUrl": "https://www.npmjs.com/package/wdio-rerun-service"
```

**Review by christian-bromann — COMMENTED** [review]
just a minor formatting issue

**Comment by mikesalvia** [comment]
Accepted and merged in.

**Review by christian-bromann — APPROVED** [review]
Thanks 👍

**Pull request merged**